frontend/tp-front/src/Pages/Home.tsx

```diff
@@ -0,0 +1,19 @@
+import Header from "../Components/Header";
+import { Typography } from 'antd';
+import { useSearchParams } from "react-router-dom";
+
+
```

Comment:
@NishedhaM An unnecessary space line.

Reply:
Resolved unnecessary space line in home page